pkg/fission-cli/cmd/function/test.go

```diff
@@ -179,6 +180,14 @@ func doHTTPRequest(ctx context.Context, url string, headers []string, method, bo
 		return nil, errors.Wrap(err, "error creating HTTP request")
 	}
 
+	if console.Verbosity >= 2 {
```

Comment:
Should we move this block to line no 203 ?  We wont be able to dump headers otherwise.
I dont see support to mask Authorization header though. I think we can dump it is.